packages/babel-preset-expo/src/index.ts

```diff
@@ -248,6 +248,10 @@ function babelPresetExpo(api: ConfigAPI, options: BabelPresetExpoOptions = {}):
     inlines['process.env.EXPO_BASE_URL'] = baseUrl;
   }
 
+  if (platform === 'web') {
+    inlines['process.env.EXPO_DOM_BASE_URL'] = baseUrl;
```

Comment:
This makes me think that we should just drop `process.env.EXPO_DOM_BASE_URL` in favor of a dynamic `process.env.EXPO_BASE_URL`.

Reply:
sounds good. do you have a recommendation where to create the dynamic `process.env.EXPO_BASE_URL`. currently the `process.env.EXPO_DOM_BASE_URL` is inline only on web. on native, it is injected by webview in runtime.